src/spec/Card.spec.jsx

```diff
@@ -0,0 +1,86 @@
+// Card.test.js
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import { describe, expect, test } from "vitest";
```

Comment:
```suggestion
import { describe, expect } from "vitest";
```
test는 현재 파일에서 사용하지 않는 것 같습니다.